### PR TITLE
Block 0 and Gen Txs

### PR DESCRIPTION
**Pull request by dutterbutter**
**Changes made in this pull request:**

- Writing block0 to db
- Writing txs from Genesis to tx table
- UI updates for accounts in place

### CODE REVIEW

File: cmd/geth/chaincmd.go [hunk 1]
```diff
@@ -168,7 +169,13 @@ func initGenesis(ctx *cli.Context) error {
 		if err != nil {
 			utils.Fatalf("Failed to open database: %v", err)
 		}
-		_, hash, err := core.SetupGenesisBlock(chaindb, genesis, nil)
+		// @NOTE:shyft instantiate BlockExplorerDB here
+		connStr := "user=postgres dbname=shyftdb sslmode=disable"
+		blockExplorerDb, err := sql.Open("postgres", connStr)
+		if err != nil {
+			return nil
+		}
```
Comment (GregTheGreek) [inline cmd/geth/chaincmd.go]: hacky but works

File: core/genesis.go [hunk 2]
```diff
@@ -156,18 +158,80 @@ func WriteShyftGen(sqldb *sql.DB, gen *Genesis) {
...
+			gasLimit := block.Header().GasLimit
 			addr := k.String()
 			txCountAccount := v.Nonce +1
+			i, err := strconv.ParseInt(block.Time().String(), 10, 64)
+			if err != nil {
+				panic(err)
+			}
+			age := time.Unix(i, 0)
+			Genesis := []string{"GENESIS_", addr}
+			GENESIS := "GENESIS"
```
Comment (GregTheGreek) [inline core/genesis.go]: Cool looks good

**Comment by GregTheGreek** [comment]
Seems to look good, also looked at it live, works as expected

**Comment by dutterbutter** [comment]
approved by Greg

**Pull request merged**